app/src/test/java/me/hufman/androidautoidrive/phoneui/MusicActivityModelTest.kt

```diff
@@ -66,4 +68,34 @@ class MusicActivityModelTest {
 		assertEquals("Exception", viewModel.errorTitle.value)
 		assertEquals("Test", viewModel.errorMessage.value)
 	}
+
+	@Test
+	fun apiAuthenticated() {
+		whenever(webApi.isUsingSpotify) doReturn false
+		whenever(webApi.isAuthorized()) doAnswer { true }
```

Comment:
You can use `doReturn` for booleans like this. `doAnswer` is mainly needed when returning a List (because `doReturn` would take the list as a sequence of responses) or other fancy mock things, like returning an anonymous mock within the mock without having to pull it out to a separate variable.

Reply:
Oh good to know, thanks! I updated all of those doAnswer to be doReturn